usmqe_tests/api/gluster/conftest.py

```diff
@@ -100,18 +125,31 @@ def invalid_volume_configuration(request):
     return request.param
 
 
-@pytest.fixture
-def valid_volume_name():
-    """
-    Generate valid volume name defined in usm.ini.
-    """
-    return pytest.config.getini("usm_volume_name")
-
-
 # TODO(fbalak) as `./,!@##$%^&*()__{}|:';/<*+>)(` as parameter
 @pytest.fixture(params=[None])
 def invalid_volume_name(request):
     """
     Generate invalid volume name.
     """
     return request.param
+
+
+@pytest.fixture
+def valid_bricks_for_crud_volume(valid_session_credentials, cluster_reuse,
+                                 valid_brick_name, valid_devices):
+
+    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
+
+    nodes = list(cluster_reuse["nodes"].values())
+
+    LOGGER.debug("nodes: {}".format(nodes))
+    LOGGER.debug("devices: {}".format(valid_devices))
+
+    job_id = api.create_bricks(
+        cluster_reuse["cluster_id"],
+        nodes,
+        valid_devices,
+        valid_brick_name)["job_id"]
+    api.wait_for_job_status(job_id)
+    import time
+    time.sleep(60)
```

Comment:
missing `yield`, I presume those bricks should be used somewhere

Reply:
It is not missing. They will be used somewhere but other fixture will get it from API call.

Reply:
What API call? Are you aware that bricks do not exist till they are not used in some volume?

Reply:
Info about bricks is in ```cluster_reuse``` fixture.

Reply:
nope, there's no such info, there is just cluster identifier no info about bricks. Cluster identifier is not enough information to get bricks.